tests/sentry/snuba/test_discover.py

```diff
@@ -531,6 +541,7 @@ def test_condition_projectname_transform(self, mock_query):
             groupby=[],
             dataset=Dataset.Discover,
             aggregations=[],
+            having=[],
```

Comment:
Would be nice to have an integration test for discover.query + having.

Reply:
This is coming once I get aliases working.